ext/mysql2/mysql2_ext.h

```diff
@@ -20,6 +20,9 @@ void Init_mysql2(void);
 #include <mysql/errmsg.h>
 #include <mysql/mysqld_error.h>
 #endif
+#ifdef HAVE_MYSQL_MYSQL_VERSION_H
+#include <mysql/mysql_version.h>
+#endif
```

Comment:
I'm not against change but please rephrase because I think I am confused.

the HAVE_MYSQL_MYSQL_VERSION_H is coming from extconf.rb from 

```ruby
have_header('mysql/mysql_version.h')
```

Thanks